static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -0,0 +1,296 @@
+    <Flex gap={space(1)} align="center">
+      <strong>
+        {hasBoth ? (
+          <Fragment>
+            {feedbackItem.user.display_name}
+            <Purple>•</Purple>
+            {feedbackItem.contact_email}
+          </Fragment>
+        ) : (
+          displayValue
+        )}
+      </strong>
+      {feedbackItem.contact_email ? (
+        <CopyToClipboardButton
+          size="xs"
+          iconSize="xs"
+          text={feedbackItem.contact_email}
+        />
+      ) : null}
+    </Flex>
+  );
+}
+
+function Viewers({feedbackItem: _}: {feedbackItem: HydratedFeedbackItem}) {
+  const displayUsers = [
+    {
+      email: 'colton.allen@sentry.io',
+      id: '1',
+      ip_address: '',
+      name: 'Colton Allen',
+      username: 'cmanallen',
+    },
+  ];
+
+  return (
+    <AvatarList
+      users={displayUsers}
+      avatarSize={28}
+      maxVisibleAvatars={13}
+      renderTooltip={user => (
+        <Fragment>
+          {userDisplayName(user)}
+          <br />
+          <DateTime date={(user as AvatarUser).lastSeen} />
+        </Fragment>
+      )}
+    />
+  );
+}
+
+function ResolveButton({feedbackItem}: {feedbackItem: HydratedFeedbackItem}) {
+  if (feedbackItem.status === 'unresolved') {
+    return (
+      <Button priority="primary" size="xs" icon={<IconArchive />}>
+        {t('Resolve')}
+      </Button>
+    );
+  }
+
+  return (
+    <Button size="xs" icon={<IconArchive />}>
+      {t('Un-Resolve')}
+    </Button>
+  );
+}
+
+const SectionWrapper = styled('section')`
+  display: flex;
+  flex-direction: column;
+  gap: ${space(3)};
+`;
+
+const SectionTitle = styled('h3')`
+  margin: 0;
+  color: ${p => p.theme.gray300};
+  font-size: ${p => p.theme.fontSizeMedium};
+  text-transform: capitalize;
+
+  display: flex;
+  gap: ${space(0.5)};
+  align-items: center;
+`;
+
+function Section({
+  children,
+  icon,
+  title,
+}: {
+  children: ReactNode;
+  title: string;
+  icon?: ReactNode;
+}) {
+  return (
+    <SectionWrapper>
+      <SectionTitle>
+        {icon}
+        <span>{title}</span>
+      </SectionTitle>
+      {children}
+    </SectionWrapper>
+  );
+}
+
+function ReplaySection({organization, replayId}) {
+  const replayPreview = useCallback(
+    () => import('sentry/components/events/eventReplay/replayPreview'),
+    []
+  );
+
+  return (
+    <Section icon={<IconPlay size="xs" />} title={t('Linked Replay')}>
+      <ErrorBoundary mini>
+        <ReplayIdCountProvider organization={organization} replayIds={[replayId]}>
+          <LazyLoad
+            component={replayPreview}
+            replaySlug={replayId}
+            orgSlug={organization.slug}
+            eventTimestampMs={0}
+            buttonProps={{
+              analyticsEventKey: 'issue_details.open_replay_details_clicked',
+              analyticsEventName: 'Issue Details: Open Replay Details Clicked',
+              analyticsParams: {
+                organization,
+              },
+            }}
+          />
```

Comment:
Wondering if it's better to move the lazy loading logic into a component so that consumers will only need to import and render said component.

Reply:
moved to ReplaySection